webapp/context.py

```diff
@@ -137,3 +138,13 @@ def get_json_feed(url, offset=0, limit=None):
         return False
 
     return content[offset:end]
+
+
+def get_user_country_by_ip():
+    client_ip = flask.request.headers.get(
+        "X-Real-IP", flask.request.remote_addr
+    )
+    ip_location = ip_reader.get(client_ip)
+    if not ip_location:
+        return None
+    return ip_location["country"]["iso_code"]
```

Comment:
@nottrobin I'm more concerned about this piece of code. Do you see any issues? How reliable is geolite2 or maybe maxminddb for that matter?

Reply:
You could probably try faking the `client_ip` variable - try picking a random address in [the public ranges](https://superuser.com/questions/1053845/what-are-the-valid-public-ip-address-ranges) and see what it does. Or, try something in one of the private ranges (`10.1.1.1` or `192.168.0.1`)

Reply:
Also curious if this will ever be an IPv6 address and how the library behaves with those.

Reply:
It seems to work equally well.